examples/DataApi-Voting-poll/scripts/send_data_transaction_create_poll.js

```diff
@@ -0,0 +1,101 @@
+const { dag4 } = require( '@stardust-collective/dag4' );
+const jsSha256 = require( 'js-sha256' );
+const jsSha512 = require( 'js-sha512' );
+const EC = require( 'elliptic' );
+const axios = require( 'axios' );
+
+const curve = new EC.ec( 'secp256k1' );
+
+/** Encode message according with serializeUpdate on your template module l1 */
+const getEncoded = ( value ) => {
+    const energyValue = JSON.stringify( value );
+    return energyValue;
+};
+
+const serialize = ( msg ) => {
+    const coded = Buffer.from( msg, 'utf8' ).toString( 'hex' );
+    return coded;
+};
+
+const sha256 = ( hash ) => {
+    return jsSha256.sha256( hash );
+};
+
+const sha512 = ( hash ) => {
+    return jsSha512.sha512( hash );
+};
+
+const sign = async ( privateKey, msg ) => {
+    const sha512Hash = sha512( msg );
+
+    const ecSig = curve.sign( sha512Hash, Buffer.from( privateKey, 'hex' ) ); //, {canonical: true});
+    return Buffer.from( ecSig.toDER() ).toString( 'hex' );
+};
+
+const sendDataTransactionsUsingUrls = async (
+    globalL0Url,
+    metagraphL1DataUrl
+) => {
+    const walletPrivateKey = ':private_key';
+
+    const account1 = dag4.createAccount();
+    account1.loginPrivateKey( walletPrivateKey );
+
+    account1.connect( {
+        networkVersion: '2.0',
+        l0Url: globalL0Url,
+        testnet: true
+    } );
+
+    const message = {
+        NewPoll: {
+            name: ':poll_name',
```

Comment:
Is `name` supposed to be ID? Can we set the ID based on the hash of the poll that's created?

Reply:
No, the name it's just the poll name. I've added the ID as well now

Reply:
The id of the poll is exactly the hash returned from the `/data` endpoint